content/contributor-guidelines.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Linebender contributor guidelines"
++++
+
+## About the projects
+
+The Linebender organization is an umbrella for a diverse and ambitious set of project, with the goal of creating high performance, accessible, high quality UI in Rust. An explicit goal is to push the boundary and fill in gaps in our understanding of UI infrastructure, as opposed to implementing existing cookbook techniques.
+
+## Discussions and community
+
+We use a [Zulip instance](https://xi.zulipchat.com/) as the primary place for discussions. All are welcome, and everyone with a GitHub account can sign up. Additional discussion happens in issues in the individual repos, and we also have tracking issues. We don’t use GitHub discussions, preferring the Zulip.
+
+We also hold weekly [office hours]. This is a chance to give status updates, make decisions, and discuss architectural directions. It is open to all, and there are public notes.
+
+We have recently re-started a [wiki], and eagerly encourage gardening. While the Zulip does have archives, they are not indexed by search engines (though search is available within Zulip), so the wiki represents the definitive place for the results of discussions and resources.
+
+To propose a nontrivial change, it is better to file an issue first rather than sending a PR. Depending on the scope and disruption of the change, it might make sense to bring it up on Zulip. Especially if a clear consensus doesn’t emerge in a reasonable time, discuss it in office hours.
+
+While the main focus of the Linebender community infrastructure is to develop Xilem, Vello, and related modules, we also want to help move the entire ecosystem forward. People from other Rust UI efforts are welcome, and discussions about how to join forces and share infrastructure especially so.
+
+We use the Rust Code of Conduct and hold people to a much higher standard of conduct than is usual on the Internet. Don’t worry if you’re interacting in good faith, though, the moderators will work with you to try to resolve issues.
+
+## Code reviews
+
+Here are our policies around code review:
+
+Every PR must be approved by someone with write access before committing.
+The [wiki] is an exception; there we optimistically merge without review
+Usually we ask the original author of the PR to commit if they have write access.
```

Comment:
```suggestion
Usually we wait for the original author of the PR to commit if they have write access.
```
As there is no asking involved I think it can help to be more precise here.